cheroot/server.py

```diff
@@ -652,7 +654,10 @@ def read_request_headers(self):
 
         # then all the http headers
         try:
-            read_headers(self.rfile, self.inheaders)
+            # Configurable using `"server.drop_underscore_headers": True`:
+#             drop_underscores = getattr(self.server.server_adapter, 'drop_underscore_headers', False)
```

Comment:
I think this comment shouldn't go into merge.
Btw, what is the difference with `server_adapter` version?